### DISCUSSION TITLE
Finally! Mantine UI went css-modules

### DISCUSSION THREAD

**Discussion by SomiDivian** [epicweb-dev/epic-stack#454] [Ideas]
I think mantine ui is the boss of ui libraries

**Comment by kentcdodds** [comment]
That's great! Care to make [an example?](https://github.com/epicweb-dev/epic-stack/blob/main/docs/examples.md)

**Reply by SomiDivian** [reply]
sure, I will soon

**Comment by SomiDivian** [comment]
I think we don't need an example

we basically need to change all the components, if the point is the integration, `mantine` has a [remix template](https://github.com/mantinedev/remix-template)

key points are:
- in `tsconfig.ts` remove the `"module"` and change the `"moduleResolution"` to `"Bundler"`
- make sure you're on `@mantine/core: 7.0.1*`

**Comment by sid374** [comment]
I'm having a hard time getting mantine to work with Remix spic stack. I keep getting typescript errors:
```
app/root.tsx:5:46 - error TS2305: Module '"@mantine/core"' has no exported member 'MultiSelect'.

5 import { MantineProvider, ColorSchemeScript, MultiSelect } from '@mantine/core'
```

I was able to somewhat fix this by doing this: https://github.com/mantinedev/mantine/issues/4991#issuecomment-1784456011

But this breaks mantine-datatable support in weird ways: https://icflorescu.github.io/mantine-datatable/
